lib/translator/translation.cc

```diff
@@ -75,7 +77,16 @@ absl::Span<const nvme_defs::GenericQueueEntryCmd> Translation::GetNvmeCmds() {
 void Translation::AbortPipeline() {
   pipeline_status_ = StatusCode::kUninitialized;
   nvme_cmd_count_ = 0;
-  // TODO free allocated PRPs
+  FlushMemory();
+}
+
+void Translation::FlushMemory() {
+  for (int i = 0; i < kMaxCommandRatio; ++i) {
```

Comment:
Should this only iterate to `nvme_cmd_count_`?

Reply:
I had it at kMaxCommandRatio just as a safety precaution in case a translation function skipped an index but as long as it's expected no indexes are skipped then using nvme_cmd_count_ is appropriate